examples/using-gatsby-image/src/pages/traced-svg.js

```diff
@@ -40,51 +40,61 @@ export default TracedSVG
 
 export const query = graphql`
   query {
-    file(relativePath: { regex: "/yoann-siloine-532511/" }) {
-      childImageSharp {
-        fluid(
-          maxWidth: 800
-          traceSVG: { background: "#fff", color: "#663399" }
-        ) {
-          ...GatsbyImageSharpFluid_tracedSVG
+    polaroid: unsplashImagesYaml(title: { eq: "Polaroid Pronto 600" }) {
+      localFile {
+        childImageSharp {
+          fluid(
+            maxWidth: 800
+            traceSVG: { background: "#fff", color: "#663399" }
+          ) {
+            ...GatsbyImageSharpFluid_tracedSVG
+          }
         }
       }
     }
-    floatingImageMobile: file(
-      relativePath: { regex: "/charles-deluvio-716558/" }
+
+    floatingImageMobile: unsplashImagesYaml(
+      title: { eq: "Pug without hoodie" }
     ) {
-      childImageSharp {
-        fixed(width: 120) {
-          ...GatsbyImageSharpFixed_tracedSVG
+      localFile {
+        childImageSharp {
+          fixed(width: 120) {
+            ...GatsbyImageSharpFixed_tracedSVG
+          }
         }
       }
     }
-    floatingImage: file(relativePath: { regex: "/charles-deluvio-716558/" }) {
-      childImageSharp {
-        fixed(width: 200) {
-          ...GatsbyImageSharpFixed_tracedSVG
+    floatingImage: unsplashImagesYaml(title: { eq: "Pug without hoodie" }) {
+      localFile {
+        childImageSharp {
+          fixed(width: 200) {
+            ...GatsbyImageSharpFixed_tracedSVG
+          }
         }
       }
     }
-    fullWidthImage: file(relativePath: { regex: "/chuttersnap-1103171/" }) {
-      childImageSharp {
-        fluid(maxWidth: 600) {
-          ...GatsbyImageSharpFluid_tracedSVG
+    fullWidthImage: unsplashImagesYaml(title: { eq: "City from above" }) {
+      credit
```

Comment:
Also @fk I don't know if we _have_ to credit, but we have it available in the YAML now 🙃

Reply:
👍 👍 🙏 Yeah that's great! We don't have to credit (https://unsplash.com/license), but I'd like to do so somehow (a little more than putting the credits in the image title). Then again, it's a demo. I think we don't even have page `<title>`s…